assessments/FastBridge/earthmover.yaml

```diff
@@ -0,0 +1,1967 @@
+version: 2
+config:
+  log_level: INFO
+  output_dir: ${OUTPUT_DIR}
+  memory_limit: 1GB
+  state_file: ${STATE_FILE}
+  show_graph: false
+  show_stacktrace: true
+  parameter_defaults:
+    # You can set these values at the command line or leave the defaults
+    OUTPUT_DIR: ./output/
+    STATE_FILE: ./tmp/.earthmover/runs.csv
+    API_YEAR: ''
+    STUDENT_ID_NAME: 'edFi_studentUniqueID' # default to the column added by the apply_xwalk package of student ID xwalking feature
+    POSSIBLE_STUDENT_ID_COLUMNS: StateID
+
+sources:
+  {% set assessment_periods = [
+    'Fall',
+    'Winter',
+    'Spring',
+    'ScreeningPeriod4',
+    'ScreeningPeriod5'
+    ] %}
+  
+  {% set objective_names = [
+    'EarlyReadingEnglish',
+    'ConceptsofPrint',
+    'DecodableWords',
+    'LetterNames',
+    'LetterSounds',
+    'NonsenseWords',
+    'OnsetSounds',
+    'OralRepetition',
+    'WordRhyming',
+    'SentenceReading',
+    'SightWords',
+    'WordBlending',
+    'WordSegmenting',
+    'CBMR-English'
+    ] %}
+
+  input:
+    file: ${INPUT_FILE}
+    header_rows: 1
+    columns:
```

Comment:
Listing all columns is only necessary if the input file is optional, otherwise we do not recommend doing so

Reply:
The reason all columns are listed is that some of the columns are calculated columns such as `Error = Total Words Read - Words read correctly`, which in turn are dependent on base periods prefix (like fall, winter, etc). I had to use column names prefix to isolate metrics for specific periods as well as objectives.